helix-view/src/editor.rs

```diff
@@ -905,6 +905,26 @@ impl Editor {
         self.tree.transpose();
     }
 
+    pub fn grow_buffer_width(&mut self) {
+        self.tree.grow_buffer_width();
+    }
+
+    pub fn shrink_buffer_width(&mut self) {
+        self.tree.shrink_buffer_width();
+    }
+
+    pub fn grow_buffer_height(&mut self) {
+        self.tree.grow_buffer_height();
+    }
+
+    pub fn shrink_buffer_height(&mut self) {
+        self.tree.shrink_buffer_height();
+    }
```

Comment:
Rather than a bunch of methods, what about one method with some enums `resize_buffer(Resize::Shrink, Dimension::Height)`